src/main/java/frc/robot/SimConstants.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot;
+
+/**
+ * This class holds all of our 2025 constants that are overrides for simulation.
+ */
+public final class SimConstants {
+
+  /** This contains sim constants for our Base Pivot. */
+  public static class SimBasePivotConstants {
+
+    public static final double kP = 400.0;
+    public static final double kI = 10.00;
+    public static final double kD = 0.0;
+    
+    public static final double canCoderOffsetDegrees = 90;
+  }
+
+  /** This contains sim constants for our Gripper Pivot. */
+  public static class SimGripperPivotConstants {
+
+    public static final double kP = 10.0;
+    public static final double kI = 0.0;
+    public static final double kD = 0.0;
```

Comment:
Most of these PID values don't need to be overridden if we can find realistic moment values for the sim (but this was a quick fix for now)